WordPress/WordPressTest/ReaderDetailCoordinatorTests.swift

```diff
@@ -350,12 +354,12 @@ private class ReaderDetailViewMock: UIViewController, ReaderDetailView {
 
 private class PostSharingControllerMock: PostSharingController {
     var didCallShareReaderPostWith: ReaderPost?
-    var didCallShareReaderPostWithView: UIView?
+    var didCallShareReaderPostWithView: PostSharingController.PopoverAnchor?
     var didCallShareReaderPostWithViewController: UIViewController?
 
-    override func shareReaderPost(_ post: ReaderPost, fromView anchorView: UIView, inViewController viewController: UIViewController) {
+    override func shareReaderPost(_ post: ReaderPost, fromAnchor anchor: PostSharingController.PopoverAnchor, inViewController viewController: UIViewController) {
```

Comment:
Types FTW 💪 👏 

Nitpick: The parameter name could be `from anchor` given that the parameter type now encodes the "anchor" information. Same for the `UIViewController` parameter.

```suggestion
    override func shareReaderPost(_ post: ReaderPost, from anchor: PostSharingController.PopoverAnchor, in viewController: UIViewController) {
```

However, I was looking for call site changes to match this rename and couldn't find any 🤔 Is the method unused?

Reply:
Oh! I checked out the code locally and understood what happened!

The method changed in `PostSharingController` but there was no compiler enforcement to remind us to update it here, too ☝️ This is one of the dangers of using subclasses instead of protocols to build test doubles: The underlying implementation can change and the test double subclass has no way of knowing it. A protocol may have a bigger footprint and require more work to setup initially, but it makes the API pact explicit, and, in a type-safe language like Swift, _enforced_ by the compiler. That is to say, this would now have happened if we had `protocol PostSharing`, `class PostSharingController: PostSharing`, and `class PostSharingSpy: PostSharing`.

@salimbraksa sorry for the ramble. It's not meant as a critique to your PR, which is actually a great sharp focused fix well suited for a release branch, where the less we change the better. I wanted to get the thought out for future reference, and maybe to build a blog post out of it.